src/features/user/Settings/ImpersonateUser/ImpersonateUser.module.scss

```diff
@@ -32,3 +32,31 @@
   max-width: 208px;
   text-align: center;
 }
+// Loader for Buttons
+.spinner {
+  border: 2px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 2px solid transparent;
+  width: 16px !important;
+  height: 16px !important;
```

Comment:
Remove `!important` for the width and height - it isn't needed here. Avoid using `!important` except in rare cases where nothing else works,the solution is very complicated (e.g. if you need to combine a large number of classes together to get enough specificity for a style to work), or a temporary urgent fix.